x-pack/plugins/uptime/public/routes.tsx

```diff
@@ -36,29 +36,42 @@ interface RouteProps {
   dataTestSubj: string;
   title: string;
   telemetryId: UptimePage;
-  pageHeader?: { pageTitle: string | JSX.Element; rightSideItems?: JSX.Element[] };
+  pageHeader?: {
+    children?: JSX.Element;
+    pageTitle: string | JSX.Element;
+    rightSideItems?: JSX.Element[];
+  };
 }
 
-const baseTitle = 'Uptime - Kibana';
+const baseTitle = i18n.translate('xpack.uptime.routes.baseTitle', {
```

Comment:
I felt that these titles should be translated, as it's static text. LMK if there's a good reason not to.